lib/api/core/models/repositories/roleRepository.js

```diff
@@ -129,28 +129,27 @@ RoleRepository.prototype.loadRole = function roleRepositoryLoadRole (roleId) {
 
 /**
  *
- * @param {Request} request
+ * @param {Array} controllers
+ * @param {Object} args
  */
-RoleRepository.prototype.searchRole = function roleRepositorySearchRole (request) {
-  var query = {query: {}};
-
-  request.input.body = request.input.body || {};
+RoleRepository.prototype.searchRole = function roleRepositorySearchRole (controllers, args) {
```

Comment:
At this point of atomification, I would also extract `size` and `from` from `args` in the function signature.